### DISCUSSION TITLE
Support 3 and 4 letter month parameters for toFormat

### DISCUSSION THREAD

**Discussion by edwardmjackson** [gastromatic/calendar-date#254] [Ideas]
Appreciate the locale difficulties here, but is there a way to support  3 and 4 letter parameters (e.g., MMM, and MMMM) for the toFormat method?

Currently, I see this:
```
(new CalendarDate("2020-07-06")).toFormat("yyyy-MMM-dd") // 2020-077-06
```

Ideally, it would return 'Jul'. 

I can do workarounds by converting to a JS Date or using MomentJS, it's just a bit messy.

**Comment by co-sic** [comment]
Hey @edwardmjackson . Sorry for the late reply, i did not see this discussion until now. I created a PR to have more formatting options: https://github.com/gastromatic/calendar-date/pull/277
